cli/jobs/pipelines-with-components/image_classification_with_densenet/image_cnn_train/main.py

```diff
@@ -569,8 +570,8 @@ def _worker_init_fn(id):
     print("started training scripts on ", socket.gethostname())
     args = parser.parse_args()
 
-    args.world_size = int(os.environ["OMPI_COMM_WORLD_SIZE"])  # node count
-    args.rank = int(os.environ["OMPI_COMM_WORLD_RANK"])  # node world rank
+    args.world_size = int(os.environ["WORLD_SIZE"])  # node count
```

Comment:
Is it possible you use another example for your case?  It seems this is the only example which demos mpi now.

Reply:
Hello @shuyums2 , @tot0 , if keep using MPI as distributed framework, is there a suggestion to change to code to accommodate the common runtime?